devices.js

```diff
@@ -381,6 +381,20 @@ const devices = [
         ],
         toZigbee: [tz.on_off],
     },
+    {
+        zigbeeModel: ['lumi.plug.mitw01'],
+        model: 'ZNCZ03LM',
+        description: 'Mi power plug ZigBee TW',
```

Comment:
What does *TW* mean?

Reply:
It means that this is the Taiwan (TW) version of Mi power plug ZigBee. AFAIK this model is Taiwan only (with a type C socket & plug).

Reply:
ok, thanks!